### ISSUE TITLE
Autocompletion when opening a new tab/window stopped working

### ISSUE THREAD

**Issue by david-caro** [vimperator/vimperator-labs#613]
##### Issue type:

- Bug report

##### Version:

```
Vimperator:	3.14.0 (created: 2016/06/07 23:58:16)
Firefox:	Mozilla/5.0 (X11; Fedora; Linux x86_64; rv:49.0) Gecko/20100101 Firefox/49.0
```

##### Description:

The autocomplete and suggestions when opening a new tab (t) or opening a url (o) stopped working with the exception of the basic bookmarks (google, bing...) but completely skips my personal bookmarks, history or already open tabs and windows.

##### Expected behavior:

It should show a list of suggestions including those from personal bookmarks, history and open tabs and windows.

##### Steps to reproduce:

Start firefox and hit 't', start typing any url that you have already entered or have bookmarked.

**Comment by david-caro** [comment]
Might be related to #538

**Comment by kylethebaker** [comment]
I am having the same issue after updating to firefox 49

```
Vimperator:	3.14.0 (created: 2016/06/07 23:58:16)
Firefox:	Mozilla/5.0 (X11; Linux x86_64; rv:49.0) Gecko/20100101 Firefox/49.0
```

**Comment by zakomo** [comment]
Adding this line to .vimperatorrc should do the trick (to me it worked):
js services.services.autoCompleteSearch.class_="@mozilla.org/autocomplete/search;1?name=unifiedcomplete"
(found on an old closed issue report)

**Comment by kylethebaker** [comment]
Thanks, zakomo. That seems to have worked me for.

**Comment by timss** [comment]
What about #590, the suggested temporary [workaround](https://github.com/vimperator/vimperator-labs/issues/590#issuecomment-248833125) or [the new release](https://github.com/vimperator/vimperator-labs/releases/tag/vimperator-3.15.0) with fixes merged (soon™ on AMO)?

**Comment by s1lvester** [comment]
Fixed here: #591

**Comment by timss** [comment]
No response from OP, but assuming this is related to #590 / #591. Reopen if that's not the case and the workaround/new release doesn't fix your issue.
